### ISSUE TITLE
Over-the-ear headphones uncomfortable for participants.

### ISSUE THREAD

**Issue by adarshp** [ml4ai/tomcat#278]
Participants report discomfort with the over-the-ear headphones - they have complained about nausea. The headphones press on the other equipment (EEG/fNIRS/eye tracker).

Potential solutions:
- Chinmai: Get rid of headphones (but keep microphone) - have the participants just talk to each other.
   - Vincent: This has a potential issue - the ASI interventions are accompanied by a ding.
- Adarsh: Get different headphones - in-ear instead of over-the-ear? We can get wireless, noise canceling ones.
  - Kobus:
    - This still leaves the issue of the microphones. Perhaps we could just get a headset with a smaller form factor.
    - We should discuss the impact of the various choices on the experiment on Wednesday.
    - Suggestion: experimenters bring their headphones/headsets to the session on Wednesday, test their:
      - comfort level
      - recording sound
      - noise cancellation.
    - Someone (Adarsh voluntold Rick) should do a bit of scouting on the web to see if there is an 'in-between' solution, like a model that is similar to our existing ones but smaller. Adarsh will send the headphone model to Rick.

**Comment by adarshp** [comment]
@rchamplin  - these are the headphones we currently have: Plantronics SHR2083-01 Rugged Noise-Canceling Headset

**Comment by eduongAZ** [comment]
@val-pf discussed that the ear-phones are difficult to clean and may gross the participants out.

**Comment by val-pf** [comment]
... and suggested we discuss as a group on Wednesday, since it's her personal opinion.
Would need to look into cleaning/sizing of in-ear headphones.

**Comment by eduongAZ** [comment]
Discussion during mission meeting:
- Rick: noise-cancelling ear-buds. Sanitation is an issue. 
- Kobus: get ear canal measuring tool. Measure ear canal while measuring cap sizes.
- Emily: besides washing the ear-pieces, give the participants the disinfectant wipes just in case they want to clean the ear-pieces themselves.
- Valeria: the mic extended from the earbud to the mouth is important because it reduces the background noises.

Consensus: get several headphones and earphones, two of each, and switch out during the pilots until we find something comfortable for the participants. Get these equipments by pilot of next Friday, April 22.

**Comment by kobus-barnard** [comment]
I don't know if we resolved the usefulness of the ear measuring idea, but in case you missed it, Paulo put a potential idea in the chat: 	https://www.3m.com/3M/en_US/p/d/v000435235/

**Comment by rchamplin** [comment]
Hi Kobus,
Yes, I did see this when Paulo put it in the chat. If we are going to try the "in the ear" headphones, this would be a good thing to have in the lab. I will look into the best place to order.
Thanks,
Rick


Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________
From: kobus-barnard ***@***.***>
Sent: Wednesday, April 13, 2022, 6:22 PM
To: ml4ai/tomcat ***@***.***>
Cc: Champlin, Rick - (rchamplin) ***@***.***>; Mention ***@***.***>
Subject: [EXT]Re: [ml4ai/tomcat] Over-the-ear headphones uncomfortable for participants. (Issue #278)


External Email

I don't know if we resolved the usefulness of the ear measuring idea, but in case you missed it, Paulo put a potential idea in the chat: https://www.3m.com/3M/en_US/p/d/v000435235/

**Comment by adarshp** [comment]
> I don't know if we resolved the usefulness of the ear measuring idea, but in case you missed it, Paulo put a potential idea in the chat: https://www.3m.com/3M/en_US/p/d/v000435235/

Apparently the minimum order is 600 via the website. @rchamplin did order one from the retail outlet for $12.

**Comment by val-pf** [comment]
04/26 tried three different types of headphones so far.

- The small in-ear ones with a stick mic connected to one bud are not good, because they fall out of the ear easily.
- The "back of the neck" headphones (bluetooth) are not good, because they obstruct EEG/fNIRS cables 
- The small ear buds with a loose mic are working, after some fine tuning of audio.
More headphones to come.

**Comment by eduongAZ** [comment]
We found that using Mumble causes echo and delay, which makes the participants uncomfortable. Tuning Mumble to deal with these issues burdens the experimenters, and does NOT enhance the performance of ASR agent.

Therefore, Rick, Adarsh, Makayla, Sadiba, and I came to the conclusion that we will not use Mumble in the experiment. We will select earphones that are not noise-canceling so that the participants can hear each other talking while also able to hear sound in Minecraft and the training videos. Their voices will be recorded by the earphone, so that audio streamer can send speech audio to the ASR agent to convert to text. 

This solution solves the voice echo and delay problem when using headphones, and that the earphones will not interfere with the goggles and the caps.

Rick ordered directional microphones so that only the speech of the target participant is recorded for speech-to-text.

**Comment by eduongAZ** [comment]
Rick and I tested the new earphones and directional microphones with positive results.

The earphones are not noise cancelling, the participants should be able to hear other people talking without requiring Mumble. It also Bluetooth connected so the there are no wires and should sit comfortably in the ears of the participants.

The directional microphone provided extra sensitivity reduction capability. Rick and I roleplayed participants playing Minecraft and talking to each other from different iMac and ASR agent is able to parse the speech separately for each iMac, i.e., no background noise was recorded by the ASR agent.

In conclusion, with new earphones and directional microphones, Mumble is no longer required and background noise are greatly filtered.

**Comment by rchamplin** [comment]
Thanks Eric for documenting this.
Rick

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Eric Duong ***@***.***>
Sent: Sunday, June 26, 2022 8:29:28 PM
To: ml4ai/tomcat ***@***.***>
Cc: Champlin, Rick - (rchamplin) ***@***.***>; Mention ***@***.***>
Subject: [EXT]Re: [ml4ai/tomcat] Over-the-ear headphones uncomfortable for participants. (Issue #278)


External Email

Rick and I tested the new earphones and directional microphones with positive results.

The earphones are not noise cancelling, the participants should be able to hear other people talking without requiring Mumble. It also Bluetooth connected so the there are no wires and should sit comfortably in the ears of the participants.

The directional microphone provided extra sensitivity reduction capability. Rick and I roleplayed participants playing Minecraft and talking to each other from different iMac and ASR agent is able to parse the speech separately for each iMac, i.e., no background noise was recorded by the ASR agent.

In conclusion, with new earphones and directional microphones, Mumble is no longer required and background noise are greatly filtered.
